app/src/main/java/org/simple/clinic/settings/changelanguage/ChangeLanguageModel.kt

```diff
@@ -0,0 +1,12 @@
+package org.simple.clinic.settings.changelanguage
+
+import org.simple.clinic.settings.Language
+
+data class ChangeLanguageModel(
+    val currentLanguage: Language?,
+    val supportedLanguages: List<Language> = emptyList()
```

Comment:
Since we aren't using the constructor outside the class and we also have a default state, can we remove the default value? This need not go in the PR, in the next on perhaps?

Reply:
Whoops, this was left by mistake. I'll remove this in the next PR.

Reply:
Removed.